package.json

```diff
@@ -188,31 +188,31 @@
   },
   "devDependencies": {
     "@actions/core": "1.10.1",
-    "@puppeteer/browsers": "1.7.1",
+    "@puppeteer/browsers": "1.8.0",
     "@rollup/plugin-commonjs": "25.0.7",
     "@rollup/plugin-node-resolve": "15.2.3",
     "@rollup/plugin-terser": "0.4.4",
-    "@types/argparse": "2.0.11",
-    "@types/chai": "4.3.8",
-    "@types/chai-as-promised": "7.1.6",
-    "@types/debug": "4.1.9",
-    "@types/mocha": "10.0.2",
-    "@types/node": "20.8.6",
+    "@types/argparse": "2.0.12",
+    "@types/chai": "4.3.9",
+    "@types/chai-as-promised": "7.1.7",
+    "@types/debug": "4.1.10",
+    "@types/mocha": "10.0.3",
+    "@types/node": "20.8.10",
     "@types/sinon": "10.0.19",
-    "@types/websocket": "1.0.7",
-    "@types/ws": "8.5.7",
-    "@typescript-eslint/eslint-plugin": "6.8.0",
-    "@typescript-eslint/parser": "6.8.0",
+    "@types/websocket": "1.0.8",
+    "@types/ws": "8.5.8",
+    "@typescript-eslint/eslint-plugin": "6.9.1",
+    "@typescript-eslint/parser": "6.9.1",
     "argparse": "2.0.1",
     "chai": "4.3.10",
     "chai-as-promised": "7.1.1",
     "chai-exclude": "2.1.0",
     "debug": "4.3.4",
-    "devtools-protocol": "0.0.1213968",
```

Comment:
@thiagowfx this should not be update like this, will open a PR to remove it from dependabot. Please don't merge.

Reply:
Just add a regex to ignore this dependency to the dependabot config